pkg/application/clusterApp.go

```diff
@@ -0,0 +1,134 @@
+package application
```

Comment:
file name should be cluster_app.go, right?

Reply:
Yes, you're right